### DISCUSSION TITLE
Postman upload test giving error

### DISCUSSION THREAD

**Discussion by lucasantoniooficial** [rebing/graphql-laravel#1077] [General]
I'm trying to test file upload in postman.

But it keeps giving this error.

Could someone give me a support so I can understand where I'm going wrong?

Query: `{"query":"mutation courseCreate($file: Upload!) {\\n    courseCreate(\\n        thumbnail: $file\\n        course_category_id: 1\\n        teacher_ids: 2\\n        name: \\"teste\\"\\n        description: \\"teste\\"\\n        price: 397.99\\n        status: \\"Publicado\\"\\n    ) {\\n        id\\n    }\\n}","variables":{"file":null}}`

![image](https://github.com/rebing/graphql-laravel/assets/58445514/f9543eeb-4d26-4218-a6cf-92fde2e3c190)

**Comment by mfn** [comment]
> ![image](https://github.com/rebing/graphql-laravel/assets/87493/cfdbfca8-edcd-41cf-80c3-6c64b4843efd)

There's a syntax error in the JSON.

The one you pasted does not look like the one you send (it contains `\\n` and stuff).

Use `dump()` or whatever debugging tool of your likely to inspect the arriving payload _or_ provide a proper dump.

Or: the problem is already the `\\n` and stuff 😅 and that's what you need to fix. As it is right now, the pasted line is *not* JSON (compatible).

**Reply by lucasantoniooficial** [reply]
I took the JSON and put it in the JSON Validator and it worked.
